client/requirements-optional.txt

```diff
@@ -0,0 +1 @@
+ray[default,data]>=2.30, <3
```

Comment:
Taking into account that we are removing `data` we are able to remove this file also, don't we @ElePT ?